plugins/main/public/controllers/management/components/management/configuration/integrations/integrations.js

```diff
@@ -38,10 +32,6 @@ const helpLinks = [
 
 const defaultIntegrations = [
   { title: 'Slack', description: 'Get alerts directly on Slack' },
-  {
-    title: 'VirusTotal',
-    description: 'Get notified when malicious software is found',
-  },
```

Comment:
**nitpick**: this is related to the configuration of the Wazuh server and this should not be modified.